elements/media-quote/src/media-quote.js

```diff
@@ -23,6 +24,241 @@ class MediaQuote extends LitElement {
+            width: 98%;
+            padding: var(--ddd-spacing-2);
+          }
+
+          @container (max-width: 1261px) and (min-width: 1000px) {
+            .quote {
+              font-size: var(--ddd-font-size-xs);
+            }
+
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+          }
+
+          @container (max-width: 999px) { /* Mobile devices */
+            .text-overlay {
+              position: relative;
+              width: 90%;
+              top: 0%;
+              left: 0%;
+              font-size: var(--ddd-font-size-xs);
+            }
+
+            .text-overlay {
+              text-align: center;
+            }
+
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+
+            figure {
+              width: 100%;
+            }
+
+            .content {
+              box-decoration-break: unset;
+              display: block;
+            }
+
+            .content, .citation {
+              padding: var(--ddd-spacing-2) var(--ddd-spacing-2);
+            }
+
+            figcaption {
+              width: 95%;
+            }
+          }
+      `,
+    ];
+  }
+
+  render() {
+    return html`
+        <div class="media-quote-container">
+          <figure>
+            <div class="top-content">
+              <div class="text-overlay"> 
+                <p class="quote">
+                  <span class="content"><slot name="quote"></slot></span>
+                    ${this.querySelector('[slot="author"]') && this.querySelector('[slot="author"]').textContent.trim().length > 0 ? html`
+                      <span class="citation">
+                        <span class="author">- <slot name="author"></slot></span> 
+                        ${this.querySelector('[slot="author-detail"]') && this.querySelector('[slot="author-detail"]').textContent.trim().length > 0 ? html`
+                          <span class="author-detail">, <slot name="author-detail"></slot></span>
+                        ` : ''}
+                      </span>
+                    ` : ''}
+                </p>  
+              </div>
+              <img src="${this.src}" alt="${this.alt}">
+            </div>
+            ${this.querySelector('[slot="caption"]') && this.querySelector('[slot="caption"]').textContent.trim().length > 0 ? html`
+              <div class="caption">
+                <div class='controller' @click=${this.controlCaption}>
+                  <p class="controller-text">
+                    <span class='show-hide'>Show Caption</span>
+                    <span class="triangle">&#9660;</span>
+                  </p>
+                </div>
+                <div class="caption-content">
+                  <figcaption><slot name="caption"></slot></figcaption>
+                </div>
+              </div>
+            ` : ''}
+          </figure>
+        </div>
+    `
+  }
+
+  controlCaption() {
```

Comment:
all this will refactor heavily once adopting the detail / summary relationship

Reply:
removed since it is no longer needed with details and summary tags